samples/snippets/submit_job.py

```diff
@@ -67,12 +67,14 @@ def submit_job(project_id, region, cluster_name):
     )
 
     print(f"Job finished successfully: {output}")
+
```

Comment:
Last I checked we didn't need to worry about line breaks at the end of the _visible_ sample so I think you can remove these two before the region tag.

Reply:
Did running the formatter (`black`) make these changes? If so I think it's okay to keep the changes.

Reply:
yes, anything that isn't in update cluster just came from running nox blacken! I'll leave them as is